community-members/camila-rangel-smith.md

```diff
@@ -0,0 +1,42 @@
+## Personas of the contributing members of _The Turing Way_
+
+* Name: **Camila Rangel Smith**
+* Pronouns: she/her
+* Date: 15 May 2020
+* Current involvement: Translation in Spanish
+
+### Bio (taken from The Alan Turing Institue)
+
+> Camila is a Research Data Scientist at The Alan Turing Institute. She holds a PhD in Particle Physics from Université Paris Diderot where she worked on the ATLAS experiment at the Large Hadron Collider at CERN. 
+> During her PhD she participated on the discovery of the Higgs Boson particle announced by CERN in 2012. 
+> She continued working on ATLAS as a postdoc with Uppsala University where she focused on searches for physics beyond the Standard Model of Particle Physics.
+
+> Right before joining the Turing, she worked as Data Scientist in the EdTech sector developing innovative products focused on the assessment process in education.
```

Comment:
I am wondering if we could use a bit simpler language. I like the content but think it might not be as accessible.

An example:
```suggestion
> Right before joining the Turing Institute, she worked on technical ways to improve how we assess learners and what they learned. 
```

Reply:
Thanks for pointing this Heidi. This was written by Camila so maybe I will combine both of your texts to keep it clear but also capture her experience as a data scientist in technical education sector. What do you think?

Reply:
:+1: